packages/frontend-api/src/Model/Cart/CartWithModificationsResult.php

```diff
@@ -132,22 +131,6 @@ public function addCartItemWithModifiedPrice(CartItem $cartItem): void
         $this->itemModifications['cartItemsWithModifiedPrice'][] = $cartItem;
     }
 
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Cart\Item\CartItem $cartItem
-     */
-    public function addCartItemWithChangedQuantity(CartItem $cartItem): void
-    {
-        $this->itemModifications['cartItemsWithChangedQuantity'][] = $cartItem;
-    }
-
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Cart\Item\CartItem $cartItem
-     */
-    public function addNoLongerAvailableCartItemDueToQuantity(CartItem $cartItem): void
-    {
-        $this->itemModifications['noLongerAvailableCartItemsDueToQuantity'][] = $cartItem;
```

Comment:
I still see `noLongerAvailableCartItemsDueToQuantity` in the generated files so you probably need to regenerate them